### PR TITLE
Add note on the Vmware to KVM instances guest OS

### PR DESCRIPTION
**Pull request by nvazquez**
Quick note extending the docs added on PR: #342 

----
📚 Documentation preview 📚: https://cloudstack-documentation--377.org.readthedocs.build/en/377/

### CODE REVIEW

**Comment by nvazquez** [comment]
@blueorangutan docbuild

File: source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst [hunk 1]
```diff
@@ -115,6 +115,8 @@ When importing a Virtual Machine from VMware to KVM, CloudStack performs the fol
 
 .. note:: As mentioned above, the migration/conversion process uses an external tool, virt-v2v, which supports most but not all the operating systems out there (this is true for both the host on which the virt-v2v tool is running as well as the guest OS of the instances being migrated by the tool). Thus, the success of the import process will, almost exclusively, depend on the success of the virt-v2v conversion. In other words, the success will vary based on factors such as the current OS version, installed packages, guest OS setup, file systems, and others. Success is not guaranteed. We strongly recommend testing the migration process before proceeding with production deployments.
 
+.. note:: The resulting imported VM is converted from VMware considering the source guest OS but it uses the default import template in CloudStack: CentOS 4/5. This does not mean that the converted VMs are CentOS VMs, it is simply a link to the default import template as it is not currently possible to select a template from which to import the VM.
```
Comment (sureshanaparti) [inline source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst:118]: is default import template - CentOS 4 or CentOS 5 (or both - picks based on guest OS)?

Comment (andrijapanicsb) [reply source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst:118]: it's not the "template" - it's the guest OS type (the template is some dummy default template)

Comment (andrijapanicsb) [reply source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst:118]: We need the same for the Import QCOW2 from Local/Shared storage (or remote KVM) - I think in all cases, the guest OS is not specified, so it defaults to CentOS 4.5

Comment (nvazquez) [reply source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst:118]: The guest OS comes from the template, and for these kind of imports we are using the default import template which has the CentOS 4.5 OS type

**Review by shwstppr — APPROVED** [review]
LGTM

File: source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst (hunk 1 above)
Comment (shwstppr) [reply source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst:118]: @sureshanaparti @andrijapanicsb are your concerns addressed?

Comment (andrijapanicsb) [reply source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst:118]: I'm not OK with that. Irrelevant of what "template" we assign the VM to (this is low level stuff that customer does not need to care about) - this is about asking user to change the "OS Type" - and we need to explain that fact - "you have to change the OS type after the import"

Comment (nvazquez) [reply source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst]: Thanks guys, fixed

**Comment by nvazquez** [comment]
@blueorangutan docbuild

**Review by andrijapanicsb — APPROVED** [review]
LGTM

**Pull request merged**